enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/resources/layer.xml

```diff
@@ -34,6 +34,21 @@
         </file>
         
         <folder name="Oracle">
+            <folder name="Assets">
+                <file name="Assets.instance">
+                    <attr name="instanceCreate" methodvalue="org.netbeans.modules.cloud.oracle.assets.RootNode.instance"/>
```

Comment:
Some leftover ? There does not seem to be `assets.RootNode` class in the PR.

Reply:
This will be used later. Commented out for now.